src/main/java/ca/corefacility/bioinformatics/irida/ria/web/admin/AdminStatisticsAjaxController.java

```diff
@@ -0,0 +1,78 @@
+package ca.corefacility.bioinformatics.irida.ria.web.admin;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * Controller to handle ajax requests for the Admin Panel statistics page.
+ */
+
+@RestController
+@RequestMapping("/ajax/statistics")
+public class AdminStatisticsAjaxController {
+
+	@Autowired
+	public AdminStatisticsAjaxController() {
+	}
+
+	/**
+	 * Get usage statistics for projects, samples, analyses, and users
+	 * for the provided time period
+	 *
+	 * @param timePeriod The time period for which to retrieve usage stats for
+	 * @return dto with usage stats
+	 */
+	@RequestMapping(value = "/basic", method = RequestMethod.GET)
+	public ResponseEntity getAdminStatistics(Long timePeriod) {
```

Comment:
Your response entities should have a type associated e.g. `ResponseEntity<String>`

Reply:
This has been updated in the basic stats and advanced stats PRs that will come after this with the actual types